src/topology/algebra/order/basic.lean

```diff
@@ -2071,6 +2071,9 @@ begin
     { exact h.2.symm ▸ tendsto_const_mul_zpow_at_top_zero h.1} }
 end
 
+-- TODO: With a different proof, this could be possibly generalised to only require a
+-- hypothetical `linear_ordered_semifield`, which would also remove the need for the
+-- `nnreal` instance of `topological_division_ring`.
```

Comment:
This instance doesn't exist. We only have [`nnreal.topological_semiring`](https://leanprover-community.github.io/mathlib_docs/topology/instances/nnreal.html#nnreal.topological_semiring) and [`nnreal.has_continuous_inv₀`](https://leanprover-community.github.io/mathlib_docs/topology/instances/nnreal.html#nnreal.has_continuous_inv%E2%82%80).